### ISSUE TITLE
The setter '_searchBarState' was called on null.

### ISSUE THREAD

**Issue by Yanaus** [bnxm/material_floating_search_bar#17]
Hello,

I tried to implement your floating search bar but the searchbar dont appear and i get this error un the compiler : 

`The setter '_searchBarState' was called on null.
`
there is my code 

```
class MapSample extends StatefulWidget {
  @override
  State<MapSample> createState() => MapSampleState();
}

class MapSampleState extends State<MapSample> {
  Completer<GoogleMapController> _controller = Completer();

  static final CameraPosition _kGooglePlex = CameraPosition(
    target: LatLng(48.866667, 2.333333),
    zoom: 14.4746,
  );

  static final CameraPosition _kLake = CameraPosition(
      bearing: 192.8334901395799,
      target: LatLng(37.43296265331129, -122.08832357078792),
      tilt: 59.440717697143555,
      zoom: 19.151926040649414);

  Map<MarkerId, Marker> markers = <MarkerId, Marker>{};

  Future _addMarkerLongPressed(LatLng latlang) async {
    setState(() {
        final MarkerId markerId = MarkerId("RANDOM_ID");
        Marker marker = Marker(
            markerId: markerId,
            draggable: true,
            position: latlang, //With this parameter you automatically obtain latitude and longitude
            infoWindow: InfoWindow(
                title: "Marker here",
                snippet: 'This looks good',
            ),
            icon: BitmapDescriptor.defaultMarker,
        );

        markers[markerId] = marker;
    });

    //This is optional, it will zoom when the marker has been created
    GoogleMapController controller = await _controller.future;
    controller.animateCamera(CameraUpdate.newLatLngZoom(latlang, 17.0));
  }


  @override
  Widget build(BuildContext context) {

    return Scaffold(
      // This is handled by the search bar itself.
      resizeToAvoidBottomInset: false,
      body: Stack(
        fit: StackFit.expand,
        children: [
          buildMap(),
          buildFloatingSearchBar(),
        ],
      ),
    );

  }

  Widget buildMap() {

    return GoogleMap(
        myLocationEnabled: true,
        myLocationButtonEnabled: true,
        initialCameraPosition: _kGooglePlex,
        onMapCreated: (GoogleMapController controller) {
          _controller.complete(controller);
        },
        onLongPress: (latlang) {
            _addMarkerLongPressed(latlang); //we will call this function when pressed on the map
            nextScreenPopup(context, Add_place(lat: latlang.latitude,long: latlang.longitude ));
        },
        markers: Set<Marker>.of(markers.values),
      );

  }

  Widget buildFloatingSearchBar() {
  final isPortrait = MediaQuery.of(context).orientation == Orientation.portrait;

  return FloatingSearchBar(
    hint: 'Search...',
    scrollPadding: const EdgeInsets.only(top: 16, bottom: 56),
    transitionDuration: const Duration(milliseconds: 800),
    transitionCurve: Curves.easeInOut,
    physics: const BouncingScrollPhysics(),
    axisAlignment: isPortrait ? 0.0 : -1.0,
    openAxisAlignment: 0.0,
    maxWidth: isPortrait ? 600 : 500,
    debounceDelay: const Duration(milliseconds: 500),
    onQueryChanged: (query) {
      // Call your model, bloc, controller here.
    },
    // Specify a custom transition to be used for
    // animating between opened and closed stated.
    transition: CircularFloatingSearchBarTransition(),
    actions: [
      FloatingSearchBarAction(
        showIfOpened: false,
        child: CircularButton(
          icon: const Icon(Icons.place),
          onPressed: () {},
        ),
      ),
      FloatingSearchBarAction.searchToClear(
        showIfClosed: false,
      ),
    ],
    builder: (context, transition) {
      return ClipRRect(
        borderRadius: BorderRadius.circular(8),
        child: Material(
          color: Colors.white,
          elevation: 4.0,
          child: Column(
            mainAxisSize: MainAxisSize.min,
            children: Colors.accents.map((color) {
              return Container(height: 112, color: color);
            }).toList(),
          ),
        ),
      );
    },
  );
}
}
```

**Comment by junaidkhan80121** [comment]
I'm facing the same problem i.e.
The setter '_searchBarState' was called on null.
Receiver: null
Tried calling: _searchBarStateInstance of 'FloatingSearchBarState'

**Comment by MohamedMoaayed** [comment]
I'm facing the same issue

**Comment by bnxm** [comment]
This should be fixed with version `0.2.4`
